README.md

```diff
@@ -417,6 +418,7 @@ to step 4.
 - Once you have removed the extraneous node_modules, Build the host project you are testing with:
    ```
    yarn dev
+   cd build
```

Comment:
Please remove this line as it is not required when using sbt. If you really want to including it, please rewrite the line below to include it.

Reply:
Actually not all the software using Docker to build directly has a build folder, so best to just remove it.

Reply:
Good point, thanks.

Reply:
You are welcome.